locale/fr.txt

```diff
@@ -0,0 +1,186 @@
+%d/%d chat messages = %d/%d messages du chat 
+%d/%d crafted = %d/%d craftés
+%d/%d deaths = %d/%d morts 
+%d/%d dug = %d/%d creusés
+%d/%d game joins = %d/%d connexions
+%d/%d placed = %d/%d placés
+%s (got) = %s (obtenu)
+%s: %s = %s : %s
+%s’s awards: = Les trophés de %s :
+(Secret Award) = (Trophé Secret) 
+<achievement ID> = <réussite ID>
+<name> = <nom>
+A Cat in a Pop-Tart?! = Un Chat dans un Pop-Tart ?!
+Achievement Unlocked! = Réussite Débloquée !
+Achievement Unlocked: = Réussite Débloquée :
+Achievement Unlocked: %s = Réussite Débloquée : %s
+Achievement not found. = Réussite non trouvée.
+All your awards and statistics have been cleared. You can now start again. = Tous vos trophés et statistiques ont été réinitialisés. Vous pouvez recommencer a nouveau.
+Aspiring Farmer = Apprenti.e. Fermière/Fermier
```

Comment:
Apparently Apprenti.e. is wrong - did you mean Apprenti?

http://www.wordreference.com/fren/Apprenti